src/Dfe.PlanTech.Web/Views/Cookies/CookiesPreferencesSetBanner.cshtml

```diff
@@ -0,0 +1,27 @@
+@using Dfe.PlanTech.Domain.Cookie;
+
+@model Dfe.PlanTech.Web.Models.CookiesViewModel
+
+@{
+    bool userPreferenceSet = TempData[CookieConstants.UserPreferenceRecordedKey] as bool? ?? false;
+}
+
+@if (userPreferenceSet)
+{
+    <div class="govuk-notification-banner govuk-notification-banner--success" role="alert"
+        aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+        <div class="govuk-notification-banner__header">
+            <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                Success
+            </h2>
+        </div>
+        <div class="govuk-notification-banner__content">
+            <p class="govuk-notification-banner__heading">
+                You’ve set your cookie preferences.
+                
+                <a class="govuk-notification-banner__link"
+                    href="@Model.ReferrerUrl">Go back to the page you were looking at</a>.
```

Comment:
This is always gonna redirect to the cookies page I think, since the journey is
-> View cookies
-> Change option and submit
-> Go to cookies page with banner with referrer = cookies page

Reply:
I've amended this to use the same JS as the back button.

This also means that it won't work with JS (obv) so I've made the default href to be `/self-assessment`

Could you have another wee test for me pelase?